app/client/server.js

```diff
@@ -1,7 +1,16 @@
 const express = require('express');
+const path = require('path');
 
 const app = express();
-app.use(express.static('./build'));
+const buildPath = path.join(__dirname, './build');
```

Comment:
these codes are just for running web on its own pod and on nodejs.
if we deploy web on api or on other types of web server like nginx, also we don't need this file.